gcs_integration/README.MD

```diff
@@ -0,0 +1,61 @@
+# Save savepoints to GCS
+
+Flink Operator supports taking savepoints and storeing it into Google Cloud Storage when deploying a job with a custom flink image with GCS connector installed. 
+
+## Create custom docker image with GCS connector
+
+Note that the job jar needs to be in the same image as GCS connector in order to save savepoints to GCS. This example supports downloading the job jar from GCS and putting it under `${FLINK_HOME}/job/` of the image, the job jar URI is set in ${FLINK_JOB_JAR_URI}. Or you can customize the entrypoint.sh or Dockerfile to upload your job jar into the custom image.
```

Comment:
There are 2 ways to provide job jar to Flink, one is to embed it in the custom image, another is to upload it to a GCS bucket and reference it in the CR with gs://my-bucket/path-to-jar.